### PR TITLE
arch: add build files

### PR DESCRIPTION
**Pull request by Craig-Proctor**
Update runtimecore branch to build

### CODE REVIEW

File: conanfile_rtc.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile
+
+
+class SPDLOGConan(ConanFile):
```
Comment (parkske) [inline conanfile_rtc.py]: Nit: SPDLogConan?

**Review by parkske — APPROVED** [review]

File: conanfile_rtc.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile
+
+
+class SPDLogConan(ConanFile):
+    name = "spdlog"
+    version = "1.9.2"
+    url = "https://github.com/Esri/spdlog/tree/runtimecore"
+    license = "https://github.com/Esri/spdlog/blob/runtimecore/LICENSE"
+    description = "Very fast, header-only/compiled, C++ logging library."
```
Comment (chri7325) [inline conanfile_rtc.py:9]: `Fast C++ logging library.` is actually what is on the about page of the library. I usually just copy paste that to stay in line with the upstream project.

**Review by chri7325 — APPROVED** [review]

File: conanfile_rtc.py (hunk 2 above)
Comment (Craig-Proctor) [reply conanfile_rtc.py:9]: I copied the first line of the [readme](https://github.com/Esri/spdlog#spdlog)

Comment (chri7325) [reply conanfile_rtc.py:9]: Ahh my bad, they usually keep the About and README the same.

**Pull request merged**